index.php

```diff
@@ -80,8 +80,13 @@
         <b>Search the Available Products: 
           <input id="gfg" type="text" 
                  placeholder="Search here">
-        </b>
-        <br>
+        </b><br><br>
```

Comment:
unnecessary tags used multiple times please remove the <br> tag wherever the tag is not useful